### PR TITLE
gofmt: run hack/verify-gofmt.sh on the entire project.

### PR DESCRIPTION
**Pull request by PI-Victor**
noticed while missing a space in my PR that while running `hack/run-gofmt` some files needed to be formatted.

it's funny how they got passed the CI in the first place?

### CODE REVIEW

File: pkg/apps/apis/apps/v1/defaults_test.go [hunk 1]
```diff
@@ -136,7 +136,7 @@ func TestDefaults(t *testing.T) {
 							DNSPolicy:                     kapiv1.DNSClusterFirst,
 							Containers: []kapiv1.Container{
 								{
-									Name: "test",
+									Name:                     "test",
```
Comment (bparees) [inline pkg/apps/apis/apps/v1/defaults_test.go]: i ran gofmt on this file and it did not produce this change

**Comment by enj** [comment]
/ok-to-test

You are likely using a different go version than CI.  Let us see what the tests say.

**Comment by PI-Victor** [comment]
i'm on go version go1.11 

you're still on go.10.* 

sorry for the noise.

dropping this, since the changes cause the CI jobs to fail.

**Pull request closed without merging**